### ISSUE TITLE
Is This Normal?

### ISSUE THREAD

**Issue by y0himba** [hassan-sd/sd-video-install#3]
I have had the batch running for over an hour.  
![y0himba-CWINDOWSsystem32cmd exe-221](https://github.com/hassan-sd/sd-video-install/assets/11462603/f113e8b9-bb1c-4e85-ad6d-2e67d31b3e57)

**Comment by havenvanheusen** [comment]
Yeah, it's just downloading the four models it needs. Each one is about 9.3 gigs but you can see about how far along it is by checking the Checkpoints folder inside the generative-models folder the batch script creates.

**Comment by Shad0wyDr3amz** [comment]
anyone know exactly how many bytes it needs to write so we all know how long it will be ,mine has been going for at least 4hrs and i have 1000mbps internet, so i would assume 40gbs wouldnt take this long to download!!!

**Comment by y0himba** [comment]
> anyone know exactly how many bytes it needs to write so we all know how long it will be ,mine has been going for at least 4hrs and i have 1000mbps internet, so i would assume 40gbs wouldnt take this long to download!!!

Read the answer above.  It really depends on how fast Huggingface sends it to you, rather than how fast your Internet is. 

On way to get them quicker is to comment the download lines out of the .bat file, and download the models manually.

Here is what I did:

After commenting out the lines containing the URLs below in the batch file, I downloaded:

https://huggingface.co/stabilityai/stable-video-diffusion-img2vid-xt/resolve/main/svd_xt_image_decoder.safetensors?download=true



https://huggingface.co/stabilityai/stable-video-diffusion-img2vid-xt/resolve/main/svd_xt.safetensors?download=true



https://huggingface.co/stabilityai/stable-video-diffusion-img2vid/resolve/main/svd_image_decoder.safetensors?download=true



https://huggingface.co/stabilityai/stable-video-diffusion-img2vid/resolve/main/svd.safetensors?download=true

..and placed them in the generative-models\Checkpoints folder.

Doing it this way took only 8 minutes at my download speed.

![image](https://github.com/hassan-sd/sd-video-install/assets/11462603/73e9fdaa-568a-4fd0-8c18-b4bac53b7ad1)
